script/dolphinscheduler-daemon.sh

```diff
@@ -33,8 +33,9 @@ echo "Begin $startStop $command......"
 
 BIN_DIR=`dirname $0`
 BIN_DIR=`cd "$BIN_DIR"; pwd`
-DOLPHINSCHEDULER_HOME=`cd "$BIN_DIR/.."; pwd`
+export DOLPHINSCHEDULER_HOME=`cd "$BIN_DIR/.."; pwd`
 
+chmod -R 700 ${DOLPHINSCHEDULER_HOME}/config
```

Comment:
Hi, why is this change needed?

Reply:
Well, I developed it on the `2.0.6-prepare` branch, this comparison is on the 2.0.5 branch